frontend/app/src/components/defi/loan/loans/makerdao/MakerDaoVaultLiquidation.vue

```diff
@@ -113,6 +114,10 @@ const totalInterestOwed = computed(() => {
   }
   return makerVault.totalInterestOwed;
 });
+
+const { getIdentifierBySymbol } = useAssetInfoRetrieval();
+
+const dai: string = getIdentifierBySymbol('DAI');
```

Comment:
can't we use it directly from data instead?

Reply:
yeah I think we can